x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/TimeseriesLifecycleType.java

```diff
@@ -40,18 +43,18 @@
     static final List<String> ORDERED_VALID_HOT_ACTIONS = Arrays.asList(SetPriorityAction.NAME, UnfollowAction.NAME, RolloverAction.NAME,
```

Comment:
I think we should add the migrate action explicitly to the hot actions as well (the hot phase doesn't have to start at `min_age: 0ms`)

Reply:
Interesting. Could this potentially cause confusion if new indices are configured to go to the hot nodes (via https://github.com/elastic/elasticsearch/pull/61342 ) and a policy has the migrate action configured to be disabled?

Reply:
It could, but I was thinking of a pre-existing index having an ILM policy applied, it would be a bit strange if only the warm/cold/frozen phases moved the data around, instead of starting it on the `hot` nodes

Reply:
Added a couple of tests for these scenarios in https://github.com/elastic/elasticsearch/pull/61377/commits/fe1fb8f5e70847ccdfb5e52e519aad6c09164e2b